draft-ietf-rtcweb-jsep.xml

```diff
@@ -1442,10 +1442,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               used for sending media, along with the mandatory "cname" source
               attribute, as specified in Section 6.1, indicating the CNAME for
               the source. The CNAME must be generated in accordance with
-              <xref target="RFC7022"></xref>. [OPEN ISSUE: How are CNAMEs
-              specified for MSTs? Are they randomly generated for each
-              MediaStream? If so, can two MediaStreams be synced?
-              See: https://github.com/rtcweb-wg/jsep/issues/4]</t>
+              <xref target="RFC7022"></xref>.
+              Each PeerConnection MUST use a different CNAME. All of the
+              MediaStreamTracks SHOULD have the same CNAME unless they have
+              different clock sources in which case the MediaStreamTracks with
+              different clocks sources MUST have different
+              CNAMEs. Future specification may define an Javascript API to set
```

Comment:
Do you mean MSTs with different clock sources on the same PC? Because clearly MSTs on different PCs should have distinct CNAMEs. As indicated above, I think this is complex enough to warrant its own section.